transport/http/config.go

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
Once we finish our release candidate cycle of rpcinit 2.0, then we can pull x/config out of x/, at which point these are all committed to as well.